docs/source/templates/autosummary/module.rst

```diff
@@ -0,0 +1,68 @@
+{{ fullname | escape | underline}}
```

Comment:
clone from https://github.com/sphinx-doc/sphinx/blob/master/sphinx/ext/autosummary/templates/autosummary/module.rst and modified.